docs/release_notes.md

```diff
@@ -5,6 +5,147 @@ hide:
+- [[PR #13053]](https://github.com/kubevirt/kubevirt/pull/13053) [0xFelix] virtctl: Users can specify a sysprep volume in VMs created with virtctl create vm
+- [[PR #12855]](https://github.com/kubevirt/kubevirt/pull/12855) [0xFelix] virtctl expose: Drop flag to set deprecated LoadBalancerIP option
+- [[PR #13008]](https://github.com/kubevirt/kubevirt/pull/13008) [0xFelix] virtctl: Allow creating a basic cloud-init config with virtctl create vm
+- [[PR #12786]](https://github.com/kubevirt/kubevirt/pull/12786) [0xFelix] virtctl: Created VMs can infer an instancetype or preference from PVC, Registry and Snapshot sources now.
+- [[PR #12557]](https://github.com/kubevirt/kubevirt/pull/12557) [codingben] Optionally create data source using virtctl image upload.
+- [[PR #13072]](https://github.com/kubevirt/kubevirt/pull/13072) [0xFelix] virtctl: virtctl create vm can now use the Access Credentials API to add credentials to a new VM
+
+### SIG-storage
+- [[PR #12355]](https://github.com/kubevirt/kubevirt/pull/12355) [alicefr] Add the volume migration state in the VM status
+- [[PR #12726]](https://github.com/kubevirt/kubevirt/pull/12726) [awels] Concurrent addvolume/removevolume using virtctl no longer fail if concurrent modifications happen
+- [[PR #12582]](https://github.com/kubevirt/kubevirt/pull/12582) [mhenriks] vmsnapshot: when checking if a VM is running, ignore runStrategy
+- [[PR #12605]](https://github.com/kubevirt/kubevirt/pull/12605) [mhenriks] vmexport: enable status subresource for VirtualMachineExport
+- [[PR #12547]](https://github.com/kubevirt/kubevirt/pull/12547) [mhenriks] virt-api: skip clone auth check when DataVolume already exists
+- [[PR #12395]](https://github.com/kubevirt/kubevirt/pull/12395) [alicefr] Add new condition for VMIStorageLiveMigratable
+- [[PR #12194]](https://github.com/kubevirt/kubevirt/pull/12194) [mhenriks] VM supports kubevirt.io/immediate-data-volume-creation: "false" which delays creating DataVolumeTemplates until VM is started
+- [[PR #12254]](https://github.com/kubevirt/kubevirt/pull/12254) [jkinred] * Reduced the severity of log messages when a `VolumeSnapshotClass` is not found. When snapshots are not enabled for a volume, the reason will still be displayed in the `status.volumeSnapshotStatuses` field of a `VirtualMachine` resource.
+- [[PR #12601]](https://github.com/kubevirt/kubevirt/pull/12601) [mhenriks] vmsnapshot: Enable status subresource for snapshot.kubevirt.io api group
+
+### SIG-network
+- [[PR #13024]](https://github.com/kubevirt/kubevirt/pull/13024) [EdDev] network binding plugin: Introduce a new `managedTap` `domainAttachmentType`
+- [[PR #13060]](https://github.com/kubevirt/kubevirt/pull/13060) [EdDev] Network binding plugins feature is declared as Beta.
+- [[PR #12613]](https://github.com/kubevirt/kubevirt/pull/12613) [orelmisan] Bridge binding: Static routes to subnets containing the pod's NIC IP address are passed to the VM.
```

Comment:
This is a bug fix related to networking.
I do not know if it needs to be left here, moved to the bug-fix section or both (like the API change ones).

Reply:
Thanks @EdDev - I'll move it to bugfixes